### DISCUSSION TITLE
Paste multiline text

### DISCUSSION THREAD

**Discussion by exlexv** [squidowl/halloy#613] [Q&A]
Hi! Like this app. I'm using it on MacOS latest version via package.

Can't find a way to send multi line text. E.g. I want to send a small piece of code of 4 lines, I have to send each line separately. In other clients there are options like `paste`. 

Probably I'm missing something. Is there something like that in `Halloy`?

**Comment by casperstorm** [comment]
Hey! I am glad you like the client.
Unfortunately we don't have a feature like that currently. If you have the time, I would suggest you to create a issue as detailed as possible with how you expect the feature to work. That way we can easier pick it up when we have time.
